task/sql-tasks.js

```diff
@@ -67,7 +91,26 @@ async function task_1_3(db) {
  *
  */
 async function task_1_4(db) {
-    throw new Error("Not implemented");
+    let result = await db.query(`
+        SELECT
+           CustomerID AS 'Customer Id',
+           count(DISTINCT OrderID) AS 'Total number of Orders',
+           ROUND(count(distinct OrderID) / (
+               SELECT
+                   count(distinct OrderID)
+               FROM
+                   OrderDetails
+           ) * 100, 5) AS '% of all orders'
+       FROM
+           Orders
+           JOIN OrderDetails USING(OrderID)

```

Comment:
We don't need to  join OrderDetails table because Orders includes all needed information.